docs/requirements.md

```diff
@@ -77,7 +77,7 @@ The storage requirements recommended on the following table **depend mainly on t
 | ---------- | ---------------------------- | ----------------------- |
 | NFS        | Yes                          | **200 GB**              |
 | RabbitMQ   | Yes                          | **8 GB**                |
-| Minio      | Yes                          | **20 GB**               |
+| Minio      | Yes                          | **40 GB**               |
```

Comment:
This has been bumped in a previous PR and we missed updating this piece of documentation

Reply:
You didn't change the default in the previous PR. You only changed our custom DO instalation, right?